invisible_cities/evm/pmaps_test.py

```diff
@@ -293,3 +298,16 @@ def test_pmt_total_energy():
         s = 0
         for pn in s12pmt.peaks: s += s12pmt.pmt_total_energy_in_peak(pn, pmt)
         assert s12pmt.pmt_total_energy(pmt) == s
+
+@mark.parametrize("args, constructor",
+     (((np.array([],dtype=np.float64), np.array([], np.float64)), Peak),
+     (({},  ), S1    ),
+     (({},  ), S2    ),
+     (({},  ), S12   ),
+     (({},{}), S2Si  ),
+     (({},{}), S1Pmt ),
+     (({},{}), S2Pmt ),
+     (({},{}), S12Pmt)))
+def test_pmap_constructors_raise_exception_when_initialized_with_empty_dicts(args, constructor):
+    with raises(InitializedEmptyPmapObject):
+        constructor(*args)
```

Comment:
Interesting choice of order: given that in Python syntax the callable precedes its arguments, I would have put the callables *before* the arguments in the parametrization.

This way around works, but it looks a bit weird.